EdFi.Ods.Utilities.Migration.Tests/appsettings.json

```diff
@@ -21,5 +21,9 @@
     "3.4": "https://www.myget.org/F/ed-fi/api/v2/package/EdFi.Ods.Populated.Template/3.4.0"
   },
   "GlendaleBackupPath": "",
-  "SqlCommandTimeout": 300
+  "SqlCommandTimeout": 300,
+  "DbDeployName": "EdFi.Suite3.Db.Deploy",
```

Comment:
I think we should create a separate section for db deploy thoughts?
```
  "DbDeploy": {
    "Version" : "2.1.0-b10040",
    "Name": "EdFi.Suite3.Db.Deploy",
    "Path": "tool",
    "Source": "https://www.myget.org/F/ed-fi"
  }
```
The benefit here is that you can create a dto with this information and bind the data to the dto. This could be a separate ticket.

Reply:
Ticket Created.